### ISSUE TITLE
→⎕LC after setting stops while tracing causes unrecoverable hang

### ISSUE THREAD

**Issue by mkromberg** [Dyalog/ride#653]
### Describe the issue you are having
→⎕LC entered from the session while tracing, after setting stops, makes the tracer disappear, but control is lost

### Did you connect to an already running interpreter or start the interpreter from RIDE?
Started 18.1.39958 under Windows

### How do you reproduce the issue?
)ed foo, create a function with a few lines of code
trace foo
once foo appears, set a stop near the end of the function (if you don't set a stop, all is well)
→⎕lc

### Paste the contents of Help → About (Shift+F1)
```
IDE:
  Version: 4.4.3474
  Platform: Win32
  Date: 2021-01-29 17:14:19 +0100
  Git commit: 9105fc792fb7f507a9b66f0265cd436d2b63565f
  Preferences:{
    "floatSingle":"0",
    "floating":"1",
    "kbdLocale":"da_DK",
    "prefixKey":"<",
    "selectedExe":"C:\\Program Files\\Dyalog\\Dyalog APL-64 18.0 Unicode\\dyalog.exe"
  }

Interpreter:
  Version: 18.1.39958
  Platform: Windows-64
  Edition: Unicode/64
  Date: Feb  2 2021 at 21:10:02
```

**Comment by e9gille** [comment]
Looks like a race condition when using floating windows.

**Comment by e9gille** [comment]
Probably causing #557

**Comment by e9gille** [comment]
Fixed by #656
